ergo/metaculus.py

```diff
@@ -125,6 +126,24 @@ def __str__(self):
             return self.data["title"]
         return "<MetaculusQuestion>"
 
+    def set_data(self, key: str, value: Any):
+        """
+        Add to data dict
+
+        :param key:
+        :param value:
+        """
+        self.data[key] = value
+
+    def add_change(self, since: int):
```

Comment:
I changed it to `calculate_change` for now, because this implies it can be used not only to introduce a change value but to update it, but unlike `set_change` it doesn't imply that the caller of this method is determining the value.

Reply:
If it were just for internal use, `set_change` might be preferable, but because this name will appear in public-facing models, I want to make sure it's easy to understand. Something like `question.calculate_change(since=yesterday)` just reads very idiomatically to me.